Gemfile

```diff
@@ -3,6 +3,7 @@ source 'http://rubygems.org'
 gem 'rails', '~> 3.2.16'
 gem 'syck', platforms: [:ruby_20]
 
+# gem 'activeadmin'
 gem 'activeadmin'
```

Comment:
@mccalluc i think one of those was pinned to a version or branch or something. But it's getting included on the following line.